### ISSUE TITLE
Loss oscilation during ERA5 training

### ISSUE THREAD

**Issue by kacpnowak** [ecmwf/WeatherGenerator#492]
### What happened?

Training loss is oscilating with high frequency as shown on the plot below

<img width="3000" height="2100" alt="Image" src="https://github.com/user-attachments/assets/dc4b8ee5-e819-4642-afdb-f47f7186f7f3" />

### What are the steps to reproduce the bug?

Train for couple of epochs 

### Version

0.0.1

### Platform (OS and architecture)

Linux

### Relevant log output

```shell

```
